src/bazel/tasks.ts

```diff
@@ -30,12 +30,23 @@ function quotedOption(option: string): vscode.ShellQuotedString {
  *
  * @param command The Bazel command to execute.
  * @param options Describes the options used to launch Bazel.
+ * @param addDefaults If the user's defaults startup and command args should be
+ *     added.
  */
 export function createBazelTask(
   command: "build" | "clean" | "test",
   options: IBazelCommandOptions,
+  addDefaults = true,
```

Comment:
Do you think we'll want to toggle this separately for build/test elsewhere, or should we just do this if `command == "clean"` and drop the argument?

Reply:
The api was generic enough, I wasn't sure if it could ever be used any place else.  Anyway, up to you.

Reply:
Let's condition it on the command for now. With this API, it's possible for the user to forget to pass the argument and then have build/test flags passed to `clean`, which is odd.

Actually, looking closer, don't we still need to support startup options for `clean`? One of the options could be `--output_base`, which would change where Bazel actually expects to clean.

Reply:
Done